clarifai.go

```diff
@@ -37,7 +37,7 @@ func NewClarifaiClient() *ClarifaiClient {
 }
 
 // Check whether the client still has access rights to Clarifai
-func (client *ClarifaiClient) isAccess() bool {
+func isCLFAccessible(client *ClarifaiClient) bool {
```

Comment:
umm why did you change that from a method of ClarifaiClient to a function? I think it makes much more sense to do clarifaiClient.isAccessible() rather than  `isCLFAccessible(clarifaiClient)`